packages/components/src/DataList/DataList.const.ts

```diff
@@ -1,2 +1,22 @@
+/**
+ *  Grab the type from breakpoints const
+ */
+type ValuesOf<T extends readonly unknown[]> = T[number];
+
+export type Breakpoints = ValuesOf<typeof BREAKPOINTS>;
+
 export const EMPTY_FILTER_RESULTS_MESSAGE = "No Results for Selected Filters";
 export const EMPTY_FILTER_RESULTS_ACTION_LABEL = "Clear Filters";
+
+/**
+ * Breakpoints that we support
+ */
+export const BREAKPOINTS = ["xs", "sm", "md", "lg", "xl"] as const;
+
+export const BREAKPOINT_SIZES: Record<Breakpoints, number> = {
+  xs: 0,
+  sm: 490,
+  md: 768,
+  lg: 1080,
+  xl: 1440,
+};
```

Comment:
doesn't have to be in this ticket but we should generate those breakpoints as a design token so we can reuse them in JS

Reply:
oh, not talking about that weird hook. talking about the CSS variables that determines the breakpoints that we document https://atlantis.getjobber.com/?path=/docs/design-breakpoints--page